flask_ldap_login/__init__.py

```diff
@@ -193,6 +209,8 @@ def direct_bind(self, username, password):
             log.debug("Binding with the BIND_DN %s" % user)
             self.conn.simple_bind_s(user, password)
         except ldap.INVALID_CREDENTIALS:
+            if self._raise_errors:
+                raise ldap.INVALID_CREDENTIALS("Unable to do a direct bind with BIND_DN %s" % user)
```

Comment:
This line should probably just be `raise` with no argument.  You do not need to create a new exception.